pinot-core/src/main/java/org/apache/pinot/core/data/manager/offline/DimensionTableDataManager.java

```diff
@@ -87,6 +87,7 @@ public static DimensionTableDataManager getInstanceByTableName(String tableNameW
   private final AtomicInteger _loadToken = new AtomicInteger();
 
   private boolean _disablePreload = false;
+  private boolean _allowDuplicatePrimaryKey = true;
```

Comment:
Using `true` as default boolean config might be error prone. Consider changing it to `errorOnDuplicatePrimaryKey` and default to `false`?

Reply:
good point. will fix